CHANGELOG.md

```diff
@@ -25,7 +25,10 @@ The types of changes are:
 * Access and erasure support for Adobe Campaign [#905](https://github.com/ethyca/fidesops/pull/905)
 
 ### Changed
+
 * Users should be able to click on the full field of a dropdown-type filter to open up the dropdown [#747](https://github.com/ethyca/fidesops/pull/903)
+* Updated the python docker base image from slim-buster to slim-bullseye [928](https://github.com/ethyca/fidesops/pull/928)
+* Removed ipython from the docker install [928](https://github.com/ethyca/fidesops/pull/928)
```

Comment:
This seems to have happened in a previous change, or not been committed here? I'm still seeing [ipython in dev-requirements.txt](https://github.com/ethyca/fidesops/blob/main/dev-requirements.txt#L3).

Reply:
I didn't look at dev-requirements. That means we had ipython in two places, it was also installed with `apt-get`. Do we want to keep it as a dev requirement? I asked in stand up and everyone said they don't use it so unless you think we should keep it I can remove it there also.

Reply:
I went ahead and removed it, but if you want it back just let me know and I'll do that.

Reply:
I use it a lot in local dev, but it's not something we need to include in the Docker containers, happy for it to go so folks can make the choice if they want it.